PromoExchange/batman
# Calculate the alterntive shipping correctly

### FIxes
- an exception being thrown from alternative shipping calculation
- Selection drop down on new purchase screen not working.
### Test Plan
1. Load any company store and compare the following with production
2. Ensure the home screen prices are the same
3. Ensure the price grid prices are the same
4. Ensure the alternative shipping drop down adjusts the price with the same value
